changelog.d/1283.bugfix.md

```diff
@@ -0,0 +1 @@
+Limit the number of possible -q and -v by ignoring exceeding values
```

Comment:
```suggestion
Fix crashes due to superfluous `-q ` and `-v` flags by discarding exceeding values
```

Reply:
superfluous `-v` flags were already discarded. Will rewrite it for `-q`